template/styles/sass/partials/_layout.scss

```diff
@@ -6,14 +6,13 @@
 }
 
 .body-content {
-    display: flex;
     position: relative;
-    overflow: hidden;
     height: 100%;
 }
 
 @media only screen and (max-width: 991px) {
     .body-content {
+        display: flex;
         flex-flow: column;
```

Comment:
Use `flex-direction: column` not flex-flow. Flex flow is a shorthand for Flex-direction and Flex-wrap, in your case, you don't have wrap so you don't need the shorthand syntax.

Reply:
Sorry, I just realize that you didn't add that line, so you are good to go.